### ISSUE TITLE
OIDC UI: DAP shows new accessibility violations

### ISSUE THREAD

**Issue by dmorgant** [OpenLiberty/open-liberty#9553]
The OIDC applications went through DAP validation in June 2019.   It passed with 3 items raised as an issue, but considered a false id.

Since then, there have been new DAP  releases that show the OIDC applications may need to be updated for accessibility.  Apparently the new DAP release focuses on the <label> tags ...  but the indicated violation help does not match the description of the violation meaning that what they show in the help as a way of fixing the violation is the way the code stands today and yet a violation is being flagged.  More work needs to be done here to sort through the new list to see if there really is anything to fix.
